src/main/java/org/openstreetmap/atlas/checks/validation/areas/AreasWithHighwayTagCheck.java

```diff
@@ -22,9 +29,13 @@
 public class AreasWithHighwayTagCheck extends BaseCheck<Long>
 {
     private static final List<String> FALLBACK_INSTRUCTIONS = Arrays.asList(
-            "Area with OSM ID {0,number,#} contains invalid highway tag.",
-            "Area with OSM ID {0,number,#} is missing area tag.");
+            "The way with OSM ID {0,number,#} has a highway value which does not match the OSM standard.",
+            "The way with OSM ID {0,number,#} has a highway value of {1}, which should not have an area=yes tag.");
     private static final long serialVersionUID = 3638306611072651348L;
+    private static final EnumSet<HighwayTag> validHighwayTags = EnumSet.of(HighwayTag.SERVICES,
+            HighwayTag.SERVICE, HighwayTag.REST_AREA, HighwayTag.PEDESTRIAN, HighwayTag.PLATFORM);
+    private static final EnumSet<HighwayTag> notToStandardHighwayTags = EnumSet
```

Comment:
nit : Is the `To` in the variable name a typo? Also constant could be in snake case.